fastlane/Fastfile

```diff
@@ -850,3 +875,62 @@ end
 def simulator_version
   '14.4'
 end
+
+def occupy_test_account
+  UI.message("Looking for a free test account...")
+  find_free_test_account
+  change_test_account_availability("T")
+end
+
+def deoccupy_test_account
+  return if $used_test_account_index == nil
+  UI.message("Freeing used test account")
+  change_test_account_availability("F")
+  $used_test_account_index = nil
+end
+
+# Returns the value of test accounts key, which is 
+# expected to look like a series of Fs and Ts.
+# Example:
+#   get_test_accounts_state
+#     => "TFFFF"
+def get_test_accounts_state
+  uri = URI.parse(UI_TESTS_ACCOUNTS_KEY)
+  response = Net::HTTP.get_response(uri)
+  accounts_state = response.body.gsub("\n","")
```

Comment:
Is this to remote the trailing `\n`?

If so, there's also [`chomp`](https://apidock.com/ruby/String/chomp) that could be used and is more "idiomatic".

```suggestion
  accounts_state = response.body.chomp
```

Reply:
Thank you, Gio! It works here!